samples/spi/ADXL345/main.cpp

```diff
@@ -34,6 +35,17 @@ int main() {
     //MOSI: CN7-3
     //MISO: CN7-2
     IO::SPI& spi = IO::getSPI<IO::Pin::PB_3, EVT::core::IO::Pin::PC_12, EVT::core::IO::Pin::PC_11>(devices, deviceCount);
+#endif
+#ifdef STM32F334x8
+    //CS: D7
+    devices[0] = &IO::getGPIO<IO::Pin::PB_10>(EVT::core::IO::GPIO::Direction::OUTPUT);
```

Comment:
To avoid conditional compilation here, you can also add the "default" SPI pins in `pin.hpp` which currently has conditional compilation for 334 vs 302 pins. That would make the samples more simple which is usually the goal

Reply:
Sounds good, are there any specific SPI pins that should be considered the "default"?

Reply:
I'd recommend using the pins that are labeled on the "Arduino" compatible portion of the STM headers, but really it can be whatever you decide.

Reply:
ok